daemon/daemon.go

```diff
@@ -923,29 +945,33 @@ func (d *daemon) grpcOpts() ([]grpc.ServerOption, error) {
 }
 
 func (d *daemon) Run() error {
+	const functionCall cError.FCall = "daemon.Run"
 
 	// start frontend
 	go func() {
 		if d.conf.TLS.Enabled {
 			// manage certificate renewal through certmagic
-			certmagic.ManageSync([]string{
+			if err := certmagic.ManageSync([]string{
 				fmt.Sprintf("*.%s", d.conf.Host.Http),
 				d.conf.Host.Grpc,
-			})
+			}); err != nil {
+				//cError.E(functionCall,err)
+			}
 			domains := []string{
 				fmt.Sprintf("*.%s", d.conf.Host.Http),
 			}
 			certmagic.HTTPPort = int(d.conf.Port.InSecure)
 			certmagic.HTTPSPort = int(d.conf.Port.Secure)
 			if err := certmagic.HTTPS(domains, d.eventPool); err != nil {
 				log.Warn().Msgf("Serving error: %s", err)
+				// todo: handle error
 			}
 
 			return
 		}
 
 		if err := http.ListenAndServe(fmt.Sprintf(":%d", d.conf.Port.InSecure), d.eventPool); err != nil {
-			log.Warn().Msgf("Serving error: %s", err)
+			fmt.Errorf("Listen and serve error %d ", err)
```

Comment:
Why replace this log?

Btw. i dont think errorf prints anything.

Reply:
this is left from another experiment, mistakenly committed change , it is log.